docs/contributing/development-community.rst

```diff
@@ -1,15 +1,37 @@
-#########################
-Development of django CMS
-#########################
+..  _development-community:
 
-django CMS is developed by a community of developers from across the world,
-with a wide range and levels of skills and expertise. Every contribution,
-however small, is valued.
+##################################
+django CMS's development community
```

Comment:
That page isn't really about contributing; it's about all the different groups and people who make up the community.